Berksfile

```diff
@@ -1,3 +1,5 @@
 source "https://supermarket.getchef.com"
 
 metadata
+
+cookbook 'config-driven-helper', git: 'https://github.com/inviqa/chef-config-driven-helper.git', ref: '1.4.1'
```

Comment:
https://supermarket.chef.io/cookbooks/config-driven-helper

Added stove and published